javers-core/src/main/java/org/javers/core/metamodel/type/TypeMapper.java

```diff
@@ -168,6 +170,33 @@ public ManagedType getJaversManagedType(GlobalId globalId){
         return getJaversManagedType(state.getClassByTypeName(typeName), expectedType);
     }
 
+    /**
+     * @throws JaversException TYPE_NAME_NOT_FOUND if given typeName is not registered
```

Comment:
TYPE_NAME_NOT_FOUND is the only exception here? If yes, then the comment needs to be refined.